roles/installer/templates/deployments/deployment.yaml.j2

```diff
@@ -363,14 +355,61 @@ spec:
 {% endif %}
           env:
 {% if development_mode | bool %}
-            - name: SDB_NOTIFY_HOST
-              valueFrom:
-                fieldRef:
-                  fieldPath: status.podIP
+              - name: SDB_NOTIFY_HOST
+                valueFrom:
+                  fieldRef:
+                    fieldPath: status.podIP
 {% endif %}
 {% if ee_extra_env -%}
-            {{ ee_extra_env | indent(width=12, first=True) }}
+              {{ ee_extra_env | indent(width=12, first=True) }}
 {% endif %}
+        - image: '{{ _image }}'
+          name: '{{ ansible_operator_meta.name }}-rsyslog'
+          imagePullPolicy: '{{ image_pull_policy }}'
+          {% if task_privileged == true %}
+          securityContext:
+            privileged: true
+          {% endif %}
+          volumeMounts:
+            - name: "{{ ansible_operator_meta.name }}-application-credentials"
+              mountPath: "/etc/tower/conf.d/execution_environments.py"
+              subPath: execution_environments.py
+              readOnly: true
+            - name: "{{ ansible_operator_meta.name }}-application-credentials"
+              mountPath: "/etc/tower/conf.d/credentials.py"
+              subPath: credentials.py
+              readOnly: true
+            - name: {{ ansible_operator_meta.name }}-settings
+              mountPath: /etc/tower/settings.py
```

Comment:
not needed as this container is not interacting with the project at a job/task level